src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -4,63 +4,107 @@ import {
   forwardRef,
   HTMLAttributes,
   useLayoutEffect,
+  useRef,
 } from 'react';
 import ReactDOM from 'react-dom';
-import { useThree } from '@react-three/fiber';
+import { GroupProps, useFrame, useThree } from '@react-three/fiber';
+import { Group, Vector3 } from 'three';
 
 // Simplified version of `drei`'s `<Html>` component
-// https://github.com/pmndrs/drei/blob/v2.2.3/src/Html.tsx
-const Html = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
-  (props, ref) => {
-    const { className, style, children, ...divProps } = props;
+// https://github.com/pmndrs/drei/blob/v6.0.3/src/web/Html.tsx
 
-    const { width, height } = useThree((state) => state.size);
-    const gl = useThree((state) => state.gl);
-    const { parentElement } = gl.domElement;
+const v1 = new Vector3();
 
-    // Container `div` for ReactDOM to render into, appended next to R3F's `canvas`
-    const [el] = useState(() => document.createElement('div'));
+export interface HtmlProps extends HTMLAttributes<HTMLDivElement> {
+  groupProps?: GroupProps;
+  followCamera?: boolean;
+  fullscreen?: boolean;
+}
 
-    useEffect(() => {
-      el.style.cssText = `position: absolute; top: 0; left: 0;`;
+const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
+  const {
+    className,
+    style,
+    children,
+    followCamera,
+    fullscreen,
+    groupProps = {},
+    ...divProps
+  } = props;
 
+  const size = useThree((state) => state.size);
+  const gl = useThree((state) => state.gl);
+  const { parentElement } = gl.domElement;
+  const camera = useThree(({ camera }) => camera);
+
+  // Container `div` for ReactDOM to render into, appended next to R3F's `canvas`
+  const [el] = useState(() => document.createElement('div'));
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const group = useRef<Group>(null!);
+
+  function updateHtmlPosition() {
+    const objectPos = v1.setFromMatrixPosition(group.current.matrixWorld);
+    objectPos.project(camera);
+    const widthHalf = size.width / 2;
+    const heightHalf = size.height / 2;
+    const pos = [
+      objectPos.x * widthHalf + widthHalf,
+      -(objectPos.y * heightHalf) + heightHalf,
+    ];
+
+    el.style.transform = `translate3d(${pos[0]}px,${pos[1]}px,0)`;
+  }
+
+  useEffect(() => {
+    el.style.cssText = `position:absolute; top:0; left:0; transform-origin:0 0;`;
+    if (followCamera) {
+      updateHtmlPosition();
+    }
+
+    if (parentElement) {
+      parentElement.append(el);
+    }
+
+    return () => {
       if (parentElement) {
-        parentElement.append(el);
+        el.remove();
       }
+      ReactDOM.unmountComponentAtNode(el);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [el, parentElement]);
```

Comment:
Wow the diff is really all other the place here. 

I changed the `useEffect` to include the part that set the `cssText` and calls `updateHtmlPosition` but nothing was changed in the clean-up function.

There is also the addition of `// eslint-disable-next-line react-hooks/exhaustive-deps`